.github/workflows/lint.yml

```diff
@@ -32,18 +32,6 @@ jobs:
         with:
           command: clippy
           args: -- -D warnings
-  readme:
```

Comment:
I'm 100% in favor of this change, but is there a reason it was done it the first place?
Are there other crates using it? If we remove it here, we should probably remove it everywhere

@haraldh @npmccallum any comment on why this is here in the first place?

Reply:
The cargo-readme check exists because the ability to import a README.md file as a doc comment via `#![doc = include_str!("../README.md")]` has only been possible since Rust 1.54. Now that we're increasing our minimum MSRV to 1.56, we can remove the cargo-readme check across all of our crates.

Reply:
I've filed https://github.com/enarx/.github/issues/32 to track this.